composer.lock

```diff
@@ -262,21 +261,7 @@
                 "uppercase",
                 "words"
             ],
-            "funding": [
```

Comment:
I think your composer version is a bit old

Reply:
Composer version 1.10.9 2020-07-16 12:57:00

Reply:
Can you try with composer 2?

Reply:
Funny, removing the vendors fixed my issue.